### ISSUE TITLE
Obscure error

### ISSUE THREAD

**Issue by springimport** [magento/magento2#6050]
### Preconditions
1. Magento CE 2.1.0 with sample data is installed.
### Steps to reproduce
1. Made a request http://example.com/admin_xxxxxx/
### Actual result

```
Warning: DOMDocument::schemaValidate(): Invalid Schema in E:\OpenServer\domains\example.com\vendor\magento\framework\Config\Dom.php on line 312

Warning: DOMDocument::schemaValidate(): Invalid Schema in E:\OpenServer\domains\example.com\vendor\magento\framework\Config\Dom.php on line 312

Fatal error: Uncaught Magento\Framework\Exception\LocalizedException: Invalid Document attribute decl. 'for', attribute 'type': The QName value '{http://www.w3.org/2001/XMLSchema}string' does not resolve to a(n) simple type definition. Line: 104 attribute decl. 'type', attribute 'type': The QName value '{http://www.w3.org/2001/XMLSchema}string' does not resolve to a(n) simple type definition. Line: 105 attribute decl. 'name', attribute 'type': The QName value '{http://www.w3.org/2001/XMLSchema}string' does not resolve to a(n) simple type definition. Line: 123 attribute decl. 'type', attribute 'type': The QName value '{http://www.w3.org/2001/XMLSchema}string' does not resolve to a(n) simple type definition. Line: 135 attribute decl. 'name', attribute 'type': The QName value '{http://www.w3.org/2001/XMLSchema}string' does not resolve to a(n) simple type definition. Line: 141 attribute decl. 'type', attribute 'type': The QName value '{http://www.w3.org/2001/XMLSchema}string' does not resolve to a(n) simple type definit in E:\OpenServer\domains\example.com\vendor\magento\framework\App\ObjectManagerFactory.php on line 284
```

The following doesn't help:

```
composer update
php -f bin/magento setup:upgrade
php -f bin/magento indexer:reindex
php -f bin/magento cache:clean
```

But it has solved the problem:

```
php -f bin/magento setup:static-content:deploy
```

I would like that developers check this place in the code, may be it a bug.

**Comment by mattcoz** [comment]
I'm getting this same exact error and nothing I try fixes it, I'm completely stuck. It just started out of nowhere.

**Comment by SaadSohailK** [comment]
I'm facing the same error, and now I'm completely sucked and unable to do any work in development mode. Any solutions?

**Comment by jakewhiteley** [comment]
Same issue with magento 2.1.7.

If I go into production mode it is fine, but now developer mode only displays the above error.

Was working fine for days, ran `setup:upgrade` and then it appeared.

**Comment by IlnitskiyArtem** [comment]
@springimport In actual result I see you use OpenServer Web server (Windows)
Magento does not support your operating system. Please confirm that the issue reproduces  on some supported system as well. Please see [Magento technology stack requirements](http://devdocs.magento.com/guides/v2.1/install-gde/system-requirements-tech.html) for more information.

**Comment by korostii** [comment]
@IlnitskiyArtem, what makes you think there is any relation between the operation system and XML validation in Magento, whatsoever?

**Comment by stuseek** [comment]
apache resart fixes it.

**Comment by JacobDrummond** [comment]
@stuseek, you've just saved me hours of debugging. Thank you!

**Comment by orlangur** [comment]
Duplicate of https://github.com/magento/magento2/issues/8542, mentioned as solved in 2.1.5 and YES it is Windows-specific.

Another possible solution: https://magento.stackexchange.com/a/134473

Please use [Magento Stack Exchange](https://magento.stackexchange.com) for questions, especially regarding unsupported operating systems.

**Comment by korostii** [comment]
>  and YES it is Windows-specific.

I still don't see what do you base that assumption on. Could be purely coincidental (unless you know exactly why the error was happening and how it got fixed).

> Please use Magento Stack Exchange for questions, especially regarding unsupported operating systems.

I wouldn't go that far. 
Magento is supposedly open-source. Even if the core team doesn't feel like investing any resources into portability (other than that via that [devbox](http://devdocs.magento.com/guides/v2.1/install-gde/docker/docker-ref.html) thing done using a docker container), I don't see anything wring if the _Community_ wants to keep the Magento cross-platform and is prepared to put some effort in that. 

As such, I would suggest marking such issues as `up for grabs` *in the future* (this issue is a duplicate report of a bug already solved, so it doesn't really matter in this particular case anymore), instead of dismissing them completely.

The `question` tag is completely inappropriate here. There aren't any questions (or even question marks, for that matter) in this issue report.
